### ISSUE TITLE
train own dataset, but Found 0 boxes for img

### ISSUE THREAD

**Issue by yfq512** [qqwweee/keras-yolo3#735]
I use 40 classes datasets to train new model by 'train_bottleneck.py' (or train.py), epochs=5, end loss=30, But use new model predict mp4 video, shows Found 0 boxes for img all the time. 
Can you help me?

**Comment by huichen98** [comment]
The problem is your voc_annotation.py file. In this file，you should modify your classes which belong to you！
